test/plots/penguin-annotated.js

```diff
@@ -0,0 +1,31 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function () {
+  const penguins = await d3.csv("data/penguins.csv", d3.autoType);
+  return Plot.plot({
+    marginLeft: 75,
+    x: {insetRight: 10},
+    marks: [
+      Plot.frame(),
+      Plot.barX(
+        penguins,
+        Plot.groupY(
+          {x: "count"},
+          {
+            y: "species",
+            fill: "sex",
+            title: "sex",
+            sort: {y: "x", reverse: true}
+          }
+        )
+      ),
+      Plot.text(["Count of penguins\ngrouped by species\n and colored by sex"], {
+        frameAnchor: "bottom-right",
+        dx: -3,
```

Comment:
looking at the image, wouldn't it be nicer if dx was a positive number to avoid the overlap with the bar?
i guess that's a bit fiddly, could we right align the text?

Reply:
hmmm, it doesn't look like this on my computer.

In fact, it's worse than this: when I see the image on the vite server, it's perfect. When I look at it in the Finder, the right alignment is broken. And on github everything is broken… Why is that???

<img width="654" alt="vite server" src="https://user-images.githubusercontent.com/7001/192586032-07c7eae7-5a21-4700-9b31-5cf6fbe3845a.png">

<img width="661" alt="Finder view" src="https://user-images.githubusercontent.com/7001/192586385-7905f94d-c895-4189-bb5b-6690e10ae4f8.png">

![Github View](https://user-images.githubusercontent.com/7001/192586689-0201f88d-dee7-485d-b800-0664089a41a4.png)

Reply:
when i download the svg and remove ```.plot tspan {
      white-space: pre;
    }```
it fixes the text anchor end issue.
i'm not seeing line-height or dx on the text marks in the svg though.

Reply:
line-height becomes the various tspan elements' *y* attributes; and dx is factored in the *x* attribute of the text element.

Apparently the faulty views are caused by the pretty formatting of the svg with beautify.html in https://github.com/observablehq/plot/blob/86789ebeb2b87f26350083dd18ed84ae4098ba0b/test/plot.js#L19 — combined with the white-space: pre; of the text elements, it creates these fake spaces.

Reply:
This side-effect of testing will be fixed after #1062.

Reply:
Now it's looking good. Thanks @enjalot for catching this!
![image](https://user-images.githubusercontent.com/7001/192969802-beb5bd6b-f1a8-41d3-ba20-9b0b6f85f51c.png)